README.md

```diff
@@ -13,3 +13,4 @@ CS 411 Project
 	SQL_PASS='PASSWORD'
 	DATABASE='DATABASE_NAME'
 	```
+3. Make sure your db is setup like in db_setup.sql (UserId and ReviewId are VARCHAR(50) instead of INTEGER)
```

Comment:
If the UserId and ReviewId are specified as varchar in the .sql file, we don't need to mention this here! Maybe just rewrite this 'Setup sql databse to match schema described in db_setup.sql'